docs/docusaurus/docs/troubleshooting.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_label: 'Troubleshooting'
+title: 'Troubleshooting'
+description: Learn more about the features and benefits of GX Cloud.
+---
+
+Use the information provided here to help you resolve issues with GX OSS or GX Cloud.
+
+### My Expectation Suite is not generating Expectations
+
+This issue is typically caused when an Expectation fails and `discard_failed_expectations` is set to `True`. To resolve this issue, set `discard_failed_expectations` to `False`.
+
+### Why did a specific Expectation fail?
+
+See [Identify failed rows in an Expectation](./guides/expectations/advanced/identify_failed_rows_expectations.md). 
+
+### How can I improve Validation performance with a large Data Asset?
+
+Use Batches and Spark. See [Request data from a Data Asset](./guides/connecting_to_your_data/fluent/batch_requests/how_to_request_data_from_a_data_asset.md) and [Connect to in-memory Data Assets - Spark](./guides/connecting_to_your_data/fluent/in_memory/connect_in_memory_data.md). 
+
+### I'm experiencing issues with my Expectations and Data Sources after upgrading GX OSS
```

Comment:
What do you think about providing a link to the Quickstart for end users to confirm they are using Fluent Data Sources? I like the Data validation workflow section that Rachel created. So something like, 

To verify the usage of latest version, please refer to "Data validation workflow" in the quickstart (link here). 

I think it'll be helpful to add that if you're using data connectors or importing RuntimeBatchRequest or BatchRequest, it's an easy signal that you're using an older GX method.